dateutil/test/test_parser.py

```diff
@@ -775,8 +775,8 @@ def test_somewhat_ambiguous_string(self):
         # The parser is choosing the wrong part for hour
         # causing datetime to raise an exception.
         dtstr = '1237 PM BRST Mon Oct 30 2017'
-        res = parse(dtstr, tzinfo=self.tzinfos)
-        assert res == datetime(2017, 10, 30, 12, 37, tzinfo=self.tzinfos)
+        res = parse(dtstr, tzinfos={"BRST": gettz("America/Chicago")})
```

Comment:
I suppose `America/Sao_Paolo` would be a better choice for this (though it doesn't really matter — and it matters less since apparently this is one of the "made up" abbreviations that has since been dropped from the tz database).